.github/workflows/nightly_vtk_master.yml

```diff
@@ -68,6 +68,8 @@ jobs:
       tbb_version: ${{ steps.set_default_versions.outputs.tbb_version }}
       usd_min_version: ${{ steps.set_default_versions.outputs.usd_min_version }}
       usd_version: ${{ steps.set_default_versions.outputs.usd_version }}
+      webp_min_version: ${{ steps.set_default_versions.outputs.webp_min_version }}
```

Comment:
do we need all these min_version here by the way ? I dont think they are used in this CI workflow

Reply:
you're right, I'll remove them